pkg/esp8266_sdk/Makefile.include

```diff
@@ -21,3 +21,13 @@ LINKFLAGS += -L$(ESP8266_SDK_BUILD_DIR)
 
 # esp8266_sdk doesn't generate any .a
 PSEUDOMODULES += esp8266_sdk
+
+# Bootloader module built from the SDK.
+DIRS += $(RIOTBASE)/pkg/esp8266_sdk/bootloader
+PSEUDOMODULES += esp_bootloader
+
+ifneq (,$(filter esp_bootloader,$(USEMODULE)))
+  # Bootloader file used by esptool.inc.mk
+  BOOTLOADER_BIN ?= $(BINDIR)/esp_bootloader/bootloader.bin
+  TEST_EXTRA_FILES += $(BOOTLOADER_BIN)
```

Comment:
There is a name clash between this an `BOOTLOADER_BIN` defined in `riotboot.mk`, can we rename it to `ESP_BOOTLOADER_BIN` (I'm not sure that is the issue though)

Also wouldn't `BOOTLOADER_BIN` need to be added to `BUILD_FILES`?

Reply:
The error message is also correct... what is the rule to generate `BOOTLOADER_BIN`?

```
make: Entering directory '/tmp/dwq.0.037298659752877716/749aff2d092a21a8cc4cbf9b746da3f6/examples/timer_periodic_wakeup'
make: *** No rule to make target '/tmp/dwq.0.037298659752877716/749aff2d092a21a8cc4cbf9b746da3f6/build/esp_bootloader/bootloader.bin', needed by 'test-input-hash'.  Stop
```

Reply:
Should the actual file to be added be `ESP_SDK_ESP_BOOTLOADER_BIN`? The whole dependency is a bit hard for me to grasp to suggest a fix for the issues on the CI...

Reply:
> There is a name clash between this an `BOOTLOADER_BIN` defined in `riotboot.mk`, can we rename it to `ESP_BOOTLOADER_BIN` (I'm not sure that is the issue though)

Hm, the variable for the bootloader binary was already called `BOOTLOADER_BIN` before this PR. Thus, I wouldn't suppose that this causes the problem. But we should think about to change it to something more specific.

> Also wouldn't `BOOTLOADER_BIN` need to be added to `BUILD_FILES`?

`BUILD_FILES` is the variable which contains all binary files that are required to flash a module without having installed the toolchain, right? 

> Should the actual file to be added be `ESP_SDK_ESP_BOOTLOADER_BIN`?

I don't think so because this variable is just used during the compilation of the bootloader.

Reply:
> Hm, the variable for the bootloader binary was already called BOOTLOADER_BIN before this PR. Thus, I wouldn't suppose that this causes the problem. But we should think about to change it to something more specific.

Yes but now its added to `TEST_EXTRA_FILES` which was not the case before, and its this addition that for some reason triggers the failed check.